frontend/components/EmptyTable/_styles.scss

```diff
@@ -43,6 +36,18 @@
       }
     }
   }
+  &__info {
```

Comment:
Undoing this as it caused bugs on other pages: https://github.com/fleetdm/fleet/issues/15998
Will double check query report page